crates/pallets/starknet/src/lib.rs

```diff
@@ -952,19 +957,19 @@ impl<T: Config> Pallet<T> {
                 parent_block_hash,
                 block_number,
                 global_state_root,
-                Felt252Wrapper::try_from(&sequencer_address).unwrap(),
+                sequencer_address,
                 block_timestamp,
                 transaction_count,
-                transaction_commitment.try_into().unwrap(),
+                Felt252Wrapper::from(transaction_commitment),
```

Comment:
no need for this once #667 is merged

Reply:
okay we can wait till that is merged then